### DISCUSSION TITLE
Why the su binary exposed when it is not mounted?

### DISCUSSION THREAD

**Discussion by FerryAr** [tiann/KernelSU#1202] [Q&A]
![Screenshot_20231210-054348_Termux](https://github.com/tiann/KernelSU/assets/38641932/a9590879-81ca-42c4-827f-e3626b8377ad)

Kernel source : https://github.com/FerryAr/kernel_xiaomi_mojito/tree/inline-ksu

**Comment by tiann** [comment]
because you grant root to the app.
